pkg/controllers/rollout/suite_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"path/filepath"
 	"testing"
 
+	fleetv1alpha1 "go.goms.io/fleet/apis/placement/v1alpha1"
```

Comment:
Also a nit, to avoid confusion, would it be better if we use the alias `placementv1alpha1`?